tests/console/consoletest_setup.pm

```diff
@@ -62,6 +62,8 @@ sub run {
         assert_script_run 'less /var/log/YaST2/y2log*|grep "Automatic DHCP configuration not started - an interface is already configured"';
     }
 
+    check_integration_services;
```

Comment:
consoletest_setup is already quite big and I recommend to not add more in it. I am trying to make it smaller, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4909 . WDYT?

Reply:
#4090 is a good idea. My use case here is to run the check in every scenario. So, I could either add the check to test module which already runs in (nearly) all scenarios, or create a separate test module which will shadow the omnipresent test module. The latter approach brings more clarity to the code, but is more error-prone as after next refactoring it may me omitted as it's non-presence won't affect anything (contrary to user serial device setup).

I am happy to rework it in the shadowy way, if it's deemed to be the future-wise approach.

Reply:
I'm not sure I understand what you mean with shadowing here. Well if you really think it is helpful to run this in all scenarios then I understand why you put it into consoletest_setup but *setup* should really just be the required parts. We already have some other test modules which are important to be scheduled in most scenarios but are not strictly necessary so yes, I would go for an explicit test module still